theano/gof/cmodule.py

```diff
@@ -1893,7 +1893,7 @@ def join_options(init_part):
         # or 64 bit and compile accordingly. This step is ignored for ARM
         # architectures in order to make Theano compatible with the Raspberry
         # Pi, and Raspberry Pi 2.
-        if not any(['arm' in flag for flag in cxxflags]) and platform.machine() != 'armv7l':
+        if not any(['arm' in flag for flag in cxxflags]) and platform.machine() not in ['armv7l','armv6l']:
```

Comment:
thanks. I think I would make this more generic: `... and not "arm" in platform.machine():`. That way, we don't need to patch it for each arm processor.

From this page, gcc do not support -m32 or -m64 for arm:

https://gcc.gnu.org/onlinedocs/gcc/Option-Summary.html#Option-Summary